src/DinosaurHydrator/DinosaurHydrator.php

```diff
@@ -2,7 +2,44 @@
 
 namespace DinoApp\DinosaurHydrator;
 
+use DinoApp\Dinosaur\Dinosaur;
+use PDO;
+
 class DinosaurHydrator
 {
+    /** Connects the database and fetches all dinosaur information, appending the foodType
+     * table with a left join and returning in an array
+     * @return Array
+     */
+    public static function getAllDinos() : Array
+    {
+        // Creates a variable which points to the correct database and gives username and password
+        $db = new PDO('mysql:host=db;dbname=dinosaurs;', 'root', 'password');
```

Comment:
Can you instead create the PDO connection outside, and pass it in.

This means you only ever need to make a single connection, that you can then just pass around to any objects that need it.